### PR TITLE
Merge pull request #3 from CMPUT-404-2023-Not-Found/dev

### PR DESCRIPTION
**Pull request by adamva**
this is a test

### CODE REVIEW

**Comment by MichaelLsh** [comment]
Sup

**Comment by MichaelLsh** [comment]
> this is a test

cool cool cool

File: README.md [hunk 1]
```diff
@@ -81,8 +81,8 @@ Create a *`.env`* file with the following content
 
 The value for *`SECRET_KEY`* can be found in our *Discord -> resources* channel
 
-Now try turning on the Django backend, then in your browser go to: *<http://localhost:8000>*
+Now try turning on the Django backend, then in your browser go to: *<http://localhost:8000>*.
```
Comment (MichaelLsh) [inline README.md:84]: Cool Cool Cool

**Comment by adamva** [comment]
hello

File: README.md (hunk 1 above)
Comment (adamva) [reply README.md:84]: boo boo

**Pull request closed without merging**